### PR TITLE
Preparing release v1.5.0

### PR DESCRIPTION
**Pull request by david-terrett**
This release updates EFRA to SOFA 2018-01-30

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,6 +34,11 @@ Bugfixes
 ERFA includes smaller changes that may or may not eventually make it into SOFA,
 addressing localized bugs or similar smaller issues:
 
+* ERFA 1.5.0 and SOFA "20180130"
+
+  + There are no differences between ERFA 1.4.0 and SOFA "20180130" except
+    for the ``eraVersion`` anderaSofaVersion`` functions added in ERFA 1.4.0.
```
Comment (timj) [inline README.rst]: Typo here with "anderaSofaVersion" and don't we mean v1.5.0?

File: RELEASE.rst [hunk 2]
```diff
@@ -22,7 +22,8 @@ Instructions for releasing ERFA
 
 * Update the version info of the shared library in the `ERFA_LIB_VERSION_INFO`
   macro of `configure.ac`. Follow the instructions in `Version numbering` below.
-  Also be sure to the ``OFA_VERSION` macro in the `configure.ac`.
+  Also be sure to the update ``SOFA_VERSION`` macro in ``configure.ac`` if
```
Comment (timj) [inline RELEASE.rst]: Extra "the"

**Comment by david-terrett** [comment]
Tim,

I fixed those typos and updated my repository and I think I have created a new pull request (or updated the existing one – I’m not quite sure).  Do you see the updated request? I’m not convinced I’ve done it right.

David

**Comment by timj** [comment]
You did update the pull request. So long as you keep pushing to the same branch name GitHub will handle it fine (you can force push to your branch as well if you need to edit commits or rebase). One oddity is the unnecessary merge branch that has turned up in the branch. Those can happen if your local branch and the remote branch are out of sync and you do a `git pull` on it.

**Comment by david-terrett** [comment]
Erik,

It is probably best if you do it. I’m not that familiar with git and if I do it there is a more than sporting chance that I mess it up which would mean even more work for you to straighten things out.

David

**Comment by eteq** [comment]
Alright, this is looking good now so I'm going to merge and then I'll do a new release momentarily.  Thanks @david-terrett for your contributions here!

**Pull request merged**